packages/orchestration/src/exos/chain-hub.js

```diff
@@ -67,15 +70,14 @@ const ChainIdArgShape = M.or(
 
 const ChainHubI = M.interface('ChainHub', {
   registerChain: M.call(M.string(), CosmosChainInfoShape).returns(),
-  getChainInfo: M.callWhen(M.string()).returns(CosmosChainInfoShape),
-  registerConnection: M.callWhen(
+  getChainInfo: M.call(M.string()).returns(VowShape),
```

Comment:
We can use this to preserve `CosmosChainInfoShape`:
https://github.com/Agoric/agoric-sdk/blob/24b0f0f8e612260ded2bdb073a76ef3d98ec476d/packages/orchestration/src/exos/chain-account-kit.js#L5
https://github.com/Agoric/agoric-sdk/blob/24b0f0f8e612260ded2bdb073a76ef3d98ec476d/packages/orchestration/src/exos/chain-account-kit.js#L42

Maybe should be moved to `@agoric/vow`?

Reply:
Thanks for pointing me to that. Looking at the implementation, that would be a looser check and still wouldn't verify the kind of Vow.
https://github.com/Agoric/agoric-sdk/blob/0cfea88fb59d27dddb8b031a5b5539127b6a305e/packages/network/src/shapes.js#L30